### PR TITLE
[WIP] Full audit of Cloud Queues RBAC matrix

### PR DESCRIPTION
**Pull request by DonNateR**
- Fixes issue #1786

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-2266f99e17/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-ae2861f5df/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-ea1228d72a/how-to/). :bow:

**Comment by kmsholcomb** [comment]
Hi, @DonNateR. Edits attached. Please let me know if you have any questions. Thanks!

[Permissions matrix for Cloud Queues_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/748374/Permissions.matrix.for.Cloud.Queues_editsKH.pdf)
[Permissions matrix for Cloud Queues_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/748373/Permissions.matrix.for.Cloud.Queues_editsKH.docx)

File: content/cloud-queues/permissions-matrix-for-cloud-queues.md [hunk 1]
```diff
@@ -1,49 +1,60 @@
...
+- **Observer** provides read-only access.
 
-Method Name | API Action | Role | Description
+The matrix displays the Cloud Queues methods grouped by category, their corresponding RESTful API commands, and the RBAC roles that are supported.
+
+### Home document operation
+
+Method | API action | Role | Description
+--- | --- | --- | ---
+Get home document |	`GET /{version}/{project_id}` |	**Observer, Creator, Admin** | Gets the home document.
```
Comment (nimbinatus) [inline content/cloud-queues/permissions-matrix-for-cloud-queues.md]: Are there extra spaces after the first `|`?

File: content/cloud-queues/permissions-matrix-for-cloud-queues.md [hunk 2]
```diff
@@ -1,49 +1,60 @@
...
+
+### Queues operations
+
+Method | API action | Role | Description
+--- | --- | --- | ---
+List queues	| `GET /{version}/{project_id}/queues` | **Observer, Creator, Admin**	| Lists queues.
```
Comment (nimbinatus) [inline content/cloud-queues/permissions-matrix-for-cloud-queues.md]: Are there extra spaces before the first and third `|`?

File: content/cloud-queues/permissions-matrix-for-cloud-queues.md [hunk 3]
```diff
@@ -1,49 +1,60 @@
...
+Create queue | `PUT /{version}/{project_id}/queues/{queue_name}` | **Admin**	| Creates a queue.
```
Comment (nimbinatus) [inline content/cloud-queues/permissions-matrix-for-cloud-queues.md]: same here before the third one

File: content/cloud-queues/permissions-matrix-for-cloud-queues.md [hunk 4]
```diff
@@ -1,49 +1,60 @@
...
+Delete queue | `DELETE /{version}/{project_id}/queues/{queue_name}` | **Admin** | Deletes the specified queue.
+Check queue existence |	`GET /{version}/{project_id}/queues/{queue_name}` |**Observer, Creator, Admin**	| Verifies whether the specified queue exists.
```
Comment (nimbinatus) [inline content/cloud-queues/permissions-matrix-for-cloud-queues.md]: space needed after the second `|`

File: content/cloud-queues/permissions-matrix-for-cloud-queues.md [hunk 5]
```diff
@@ -1,49 +1,60 @@
...
+### Message operations
+
+Method | API action | Role | Description
+--- | --- | --- | ---
+Post messages | `POST /{version}/{project_id}/queues/{queue_name}/messages` | **Creator, Admin** | Posts the message or messages for the specified queue.
+Get messages | `GET /{version}/{project_id}/queues/{queue_name}/messages` | **Creator, Admin** | Gets the message or messages in the specified queue.
+Get messages by ID | `GET /{version}/{project_id}/queues/{queue_name}/messages/{messageId}` | **Observer, Creator, Admin** | Gets the specified set of messages from the specified queue.
+Bulk-delete messages by ID | `DELETE /{version}/queues/{queue_name}/messages` | **Admin** | Deletes the specified messages from the specified queue.
+Show message details | `GET /{version}/{project_id}/queues/{queue_name}/messages/{messageId}` | **Observer, Creator, Admin** | Shows details for the specified message from the specified queue.
+Delete message | `DELETE /{version}/{project_id}/queues/{queue_name}/messages/{messageId}` | **Admin**	| Deletes the specified message from the specified queue.
```
Comment (nimbinatus) [inline content/cloud-queues/permissions-matrix-for-cloud-queues.md]: extra space before the third `|`?

File: content/cloud-queues/permissions-matrix-for-cloud-queues.md [hunk 6]
```diff
@@ -1,49 +1,60 @@
...
-
-### Cloud Queues Terminology
-
-- **Message** - A task, a notification, or any meaningful data that gets posted to the queue. A message exists until it is deleted by a recipient or automatically by the system based on a TTL (time-to-live) value.
-
-- **Queue** - A queue holds messages. Ideally, a queue is created per work type. For example, if you want to compress files, you would create a queue dedicated to this job. Any application that reads from this queue would only compress files.
-
-**Related article** - [Permission Matrices for RBAC](/how-to/permissions-matrix-for-role-based-access-control-rbac)
+Claim messages | `POST /{version}/{project_id}/queues/{queue_name}/claim` | **Creator, Admin** | Claims a set of messages from the specified queue.
+Query claim	| `GET /{version}/{project_id}/queues/{queue_name}/claims/{claimId}` | **Observer, Creator, Admin** |	Queries the specified claim for the specified queue.
```
Comment (nimbinatus) [inline content/cloud-queues/permissions-matrix-for-cloud-queues.md]: extra space before the first one?

File: content/cloud-queues/permissions-matrix-for-cloud-queues.md [hunk 7]
```diff
@@ -1,49 +1,60 @@
...
-Update Claim | ```PATCH/ {version}/queues/{queue_name}/claims/{claimId}``` | **Creator & Admin** | Updates the specified claim for the specified queue.
-Release Claim | ```DELETE/ {version}/queues/{queue_name}/claims/{claimId}``` | **Creator & Admin** | Releases the specified claim for the specified queue.
-
-### Cloud Queues Terminology
-
-- **Message** - A task, a notification, or any meaningful data that gets posted to the queue. A message exists until it is deleted by a recipient or automatically by the system based on a TTL (time-to-live) value.
-
-- **Queue** - A queue holds messages. Ideally, a queue is created per work type. For example, if you want to compress files, you would create a queue dedicated to this job. Any application that reads from this queue would only compress files.
-
-**Related article** - [Permission Matrices for RBAC](/how-to/permissions-matrix-for-role-based-access-control-rbac)
```
Comment (nimbinatus) [inline content/cloud-queues/permissions-matrix-for-cloud-queues.md]: Any reason for deleting the link to the general RBAC matrix article?

**Comment by DonNateR** [comment]
Can't see the spacing issue in Atom or the GitHub editing UI. However, the spacing shouldn't effect the formatting of the tables.

**Comment by nimbinatus** [comment]
Maybe the spacing thing is a glitch in preview. LGTM, then. Ready to merge.

**Pull request merged**